kolla/defaults.yml

```diff
@@ -31,14 +31,15 @@ switch_configs: []
 neutron_networks: []
 
 enable_mariadb: yes
+enable_mariabackup: yes
```

Comment:
are there any defaults that need to be set if this is auto-enabled?

Reply:
I don't think so.